extensions/neo4j/runtime/src/main/java/io/quarkus/neo4j/runtime/Neo4jConfiguration.java

```diff
@@ -46,4 +53,57 @@
         @ConfigItem(defaultValue = "false")
         public boolean disabled = false;
     }
+
+    @ConfigGroup
+    static class Pool {
+
+        /**
+         * Flag, if metrics are enabled.
+         */
+        public boolean metricsEnabled = false;
```

Comment:
Why no `@ConfigItem` here?

Reply:
Habit. Not used to the fact that I _have_ to add the annotation, was expecting plain bean mapping in such cases. Maybe it should be added to https://quarkus.io/guides/extension-authors-guide#configuration like "Leaf configuration keys must use `@ConfigItem`…"

I'll fixe it.